test/extended/deployments/deployments.go

```diff
@@ -783,10 +782,7 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 				o.Expect(fmt.Errorf("expected no deployment, found %#v", rcs[0])).NotTo(o.HaveOccurred())
 			}
 
-			_, err = updateConfigWithRetries(oc.AppsClient().Apps(), oc.Namespace(), dcName, func(dc *appsv1.DeploymentConfig) {
```

Comment:
@tnozicka not sure what the intention here was, but if we want to patch then lets patch ;-)